ansible/roles/ami_9_aarch64/tasks/bootloader.yaml

```diff
@@ -35,7 +35,9 @@
 
 - name: Generate new GRUB environment block
   ansible.builtin.command:
-    cmd: chroot /rootfs grub2-editenv -v - set saved_entry=ffffffffffffffffffffffffffffffff-{{ kernel_ver.stdout }}
+    cmd: >
+      chroot /rootfs grub2-editenv -v - set
```

Comment:
Is there any reason you're not passing along `kernelopts="{{ ami_9_x86_64_kernel_opts }}` for EL9? Is there any downside/negative to specifying the UUID of the root partition in AL9 images, too?

Reply:
The AL9 doesn't need it inside the GRUB Environment Block (`grubenv`).